WebService/ComparisonEngineUI/ComparisonEngineUI/ViewModels/BarViewModel.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using ComparisonEngineUI.Views;
+using Xamarin.Forms;
+using ComparisonEngineUI.Models;
+using ComparisonEngineUI.Data;
+
+namespace ComparisonEngineUI.ViewModels
+{
+    
+    public class BarViewModel : BaseViewModel
+    {
+        private ListContainer listContainer = ListContainer.Instance;
+        public Bar bar;
+        private string _Name;
+        public string Name
+        {
+            get
+            {
+                return _Name;
+            }
+            set
+            {
+                _Name = value;
+                OnPropertyChanged();
+            }
```

Comment:
why do we need the private _Name, when it can be publicly get through public Name, in that case it don't make sense to have it